applications/ConvectionDiffusionApplication/custom_utilities/bfecc_convection.h

```diff
@@ -51,10 +51,10 @@ template<std::size_t TDim> class BFECCConvection
 
     //**********************************************************************************************
     //**********************************************************************************************
-    void BFECCconvect(ModelPart& rModelPart, const Variable< double >& rVar, const Variable<array_1d<double,3> >& conv_var, const double substeps)
+    void BFECCconvectPartially(ModelPart& rModelPart, const Variable< double >& rVar, const Variable<array_1d<double,3> >& conv_var, const double substeps, const double dt_factor)
```

Comment:
```suggestion
    void BFECCConvectPartially(
        ModelPart& rModelPart,
        const Variable< double >& rVar,
        const Variable<array_1d<double,3> >& conv_var,
        const double substeps,
        const double DtFactor)
```
Please follow [style guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide) for modified lines and new variables.

Just to be sure, the old `BFECCconvect` is kept right?

Reply:
Yes, the original "BFECCconvect" is working fine.